CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change log
 
+## 0.1.3 (2017-11-27) ([@TheSmartnik][])
```

Comment:
You have to add a link to the end of the file in order it to be resolvable

Reply:
And let's use `master` version and do not update the version itself.

Reply:
> You have to add a link to the end of the file in order it to be resolvable

Thanks 👍  Didn't know that